Update service catalog troubleshooting.md (#18472)
DataDog/documentation: content/en/tracing/service_catalog/troubleshooting.md

{{< img src="tracing/service_catalog/svc_cat_troubleshooting_1.png" alt="Hover message indicating that no performance data has been reported in the past hour" >}}

## SLOs not listed in Setup Guidance section

> The count in the Service Catalog Setup Guidance section reflects the number of SLOs with `service` tags. If your SLOs are not listed, you can first verify if they have `service` tag values specified and matched with the service names in other product (e.g., APM, USM). 

Reviewer: ```suggestion
The count in the Service Catalog Setup Guidance section reflects the number of SLOs with `service` tags. If your SLOs are not listed, verify that they have `service` tag values specified and that they match with the service names in other products such as APM and USM.
```
